### DISCUSSION TITLE
Codium + avx2

### DISCUSSION THREAD

**Discussion by MorviNord** [Alex313031/codium#13] [General]
I saw there was a version with avx, now I have it installed on Pop OS. Will there be avx2 and the next update with avx?

Very pleased, brother. Do you have a TON crypto wallet? I'll send some for coffee.

**Comment by Alex313031** [comment]
@MorviNord I don't have any crypto. Just Patreon and Paypal (which you can see in the donate button in alot of my repos).

And well, the AVX version compiles stuff with AVX, but crucially, it downloads and uses my own custom AVX builds of Electron 22 from here > https://github.com/Alex313031/electron-22

So, an AVX2 build would compile some things for AVX2, but the majority of the speed increase comes from the compiler optimized Electron. So I would have to make AVX2 builds of Electron 22 for all the platforms, and I don't really feel like doing that because that was a shitshow the first time around to get it to work with AVX.
